packages/token-search-discovery-controller/src/token-discovery-api-service/token-discovery-api-service.test.ts

```diff
@@ -124,4 +124,26 @@ describe('TokenDiscoveryApiService', () => {
       expect(results).toStrictEqual(mockTrendingResponse);
     });
   });
+
+  describe('getTopGainersByChains', () => {
+    it('should return top gainers results', async () => {
+      nock(TEST_API_URLS.PORTFOLIO_API)
+        .get('/tokens-search/top-gainers-by-chains')
+        .reply(200, mockTrendingResponse);
+
+      const results = await service.getTopGainersByChains({});
```

Comment:
Noticing here that the argument can be an empty object and the function will perform as expected.

Maybe we could make the argument entirely optional ?

Reply:
that's a good idea, and callers would mostly want to call it with defaults.  I'll make the change.